Bio/PDB/PDBList.py

```diff
@@ -256,6 +256,9 @@ def retrieve_pdb_file(
         :param pdir: put the file in this directory (default: create a PDB-style directory tree)
         :type pdir: string
 
+        :param compress: if set to True, existing structure files will be gzip stored. Default: False
```

Comment:
The meaning of this isn't immediately clear to me, maybe something like "downloaded files will be gzip stored"?

Reply:
Yes, is much clear like you said.